Orange/widgets/classify/owclassificationtreegraph.py

```diff
@@ -363,10 +366,21 @@ def split_condition(self):
         :return: split condition to reach a particular node.
         """
         if self.i > 0:
-            sign = [">", "<="][self.tree.children_left[self.parent.i] == self.i]
-            thresh = self.tree.threshold[self.parent.i]
-            return "%s %s" % (
-                sign, self.domain.attributes[self.attribute()].str_val(thresh))
+            attribute = self.domain.attributes[self.attribute()]
+            parent_attr = self.domain.attributes[self.parent.attribute()]
+            parent_attr_cv = parent_attr.compute_value
+            is_left_child = self.tree.children_left[self.parent.i] == self.i
+            if parent_attr_cv and isinstance(parent_attr_cv, Indicator) and \
```

Comment:
The first check in `if parent_attr_cv and isinstance(parent_attr_cv, Indicator)` is unnecessary and included in the second part (None is not an instance of Indicator)

(similarly below)